### PR TITLE
Add meeting notes for 2019-10-29

### PR DESCRIPTION
**Pull request by klappradla**
Here's the notes for today's meeting. Please check if there's anything wrong or missing (and feel free to edit).

### CODE REVIEW

File: meetings/2019-10-29.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## Decide on workshop
+
+### Which audience
+
+- start with intermediate:
+  - works best for first workshop
+  - quick vote: we start with intermediate
+- "intermediate" means confident with with computers
+  - this of course includes juniors
```
Comment (PragTob) [inline meetings/2019-10-29.md:35]: I'd add that we'd also take more experienced people that just haven't done elixir before (Mid/Senior+)

File: meetings/2019-10-29.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  - early February (Saturday 1st of February)
+- we need 6 weeks for preparation
+
+## Outcome
+
+- **Nicolo** will set up Doodle for next meetup
+- do an intermediate workshop with the snake tutorial
+  - write tutorial for attendees setting up snake game from scratch (using `scenic_new`)
+  - **Max, Nico and Markus** meet on Monday 4th to start with the guide (flash out structure, write first chapter)
+- **Tobi** is doing a general intro guide to Elixir (what's necessary for the snake tutorial)
```
Comment (PragTob) [inline meetings/2019-10-29.md]: I didn't explicitly say it, but unless someone needs it earlier I'd likely do this in December as I probably won't have a project in december aka lots of time. I might start earlier though :)

**Review by PragTob — COMMENTED** [review]
Looking good to me, only small comments :)

IMG_20180317_110254

File: meetings/2019-10-29.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Code Curious Elixir Meeting 2019-10-29
+
+## Attendees
+
+- Jens (moderating)
+- Nicolo
+- Markus
+- Max
+- Tobi
+- Daniel
```
Comment (jvf) [inline meetings/2019-10-29.md]: Should be Gui (@ggpasqualino), not Daniel

File: meetings/2019-10-29.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+- start with intermediate:
+  - works best for first workshop
+  - quick vote: we start with intermediate
+- "intermediate" means confident with with computers
+  - this of course includes juniors
+
+### Which content
+
+- recap:
+  - nerves project (we stopped looking at it)
```
Comment (jvf) [inline meetings/2019-10-29.md]: "stopped looking at it" seems wrong, we always talked about it but we did not have any further input and work done on it.

File: meetings/2019-10-29.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+- round of votes:
+  - snake fun and entertaining
+  - snake lots of Elixir good to start
+  - snake has visual aspects
+  - snake tutorial has "natural steps" (works great for guide)
+  - snake does not work for finding a job, but valid for tutorial
+  - hardware project: makes sense for the future, but too much orga effort for now
+  - elixir girls app: we would need to update the guide (no ETS, less JS)
+    - pro: great for lots of elixir and no phoenix
+    - pro: has lots of the language features in it
```
Comment (jvf) [inline meetings/2019-10-29.md:55]: Is it possible that these two points are referring to the Snake Tutorial?

**Review by jvf — CHANGES_REQUESTED** [review]

File: meetings/2019-10-29.md (hunk 3 above)
Comment (klappradla) [reply meetings/2019-10-29.md]: Haha, oh my god 🙈 So sorry @ggpasqualino 🙇 (I actually don't even know why I came up with that in the first place...)

File: meetings/2019-10-29.md (hunk 5 above)
Comment (klappradla) [reply meetings/2019-10-29.md:55]: In theory yes, but they were mentioned when referring to the Slack clone tutorial.

File: meetings/2019-10-29.md (hunk 4 above)
Comment (klappradla) [reply meetings/2019-10-29.md]: 👍 rephrased it.

File: meetings/2019-10-29.md (hunk 1 above)
Comment (klappradla) [reply meetings/2019-10-29.md:35]: Added a second note for that.

File: meetings/2019-10-29.md (hunk 2 above)
Comment (klappradla) [reply meetings/2019-10-29.md]: Added a note for that.

**Review by nickgnd — APPROVED** [review]
👏

**Review by jvf — APPROVED** [review]

**Pull request merged**